Cargo.toml

```diff
@@ -11,15 +11,14 @@ edition = "2018"
 [dependencies]
 regex = "1.3"
 lazy_static = "1.4.0"
-rocket = "0.4.5"
-rocket_codegen = "0.4.5"
-miri = { git = "https://github.com/rust-lang/miri.git", rev = "c641fbde0223cde9fae67de3cc4892998a21cec0" }
+rocket = "0.4.7"
+# This is the version of miri for the nightly in the rust-toolchain file. This might be superfluous
```

Comment:
```suggestion

```
It has to match.

Reply:
We should probably have a comment to that effect then?

Reply:
```suggestion
# This is the version of miri for the nightly in the rust-toolchain file.
```
?

Reply:
I've updated the comment text, although if you'd still prefer to just remove it we can do that too